src/classes/CRLP_RollupUI_SVC.cls

```diff
@@ -857,6 +859,13 @@ public with sharing class CRLP_RollupUI_SVC {
                 fieldObjectList.add(fieldObject);
             }
         }
+        /*FOR TESTING*/
```

Comment:
this is in here for @dgamboni to complete testing on the warning toast when no fields are available. We'll need to create a break out bug for when the mode isn't create, but the rest is ready for testing.